website/events/models/event.py

```diff
@@ -247,6 +247,41 @@ def cancellation_allowed(self):
             and self.registration_start <= now < self.start
         )
 
+    @property
+    def present_registrations(self):
+        return self.participants.filter(present=True)
+
+    @property
+    def unpresent_registrations(self):
+        return self.participants.filter(present=False)
+
+    @property
+    def unpaid_registrations(self):
```

Comment:
This is only used in `everything_paid` I believe, so it could be inside that function instead making the code more concise.

Reply:
I think this is better as it separates the different logic steps better